core/lib/utils/string_convert.dart

```diff
@@ -0,0 +1,10 @@
+class StringConvert {
+ static String? writeEmptyToNull(String text) {
```

Comment:
IMO, you should use `extension`

Reply:
Some cases extension will not working with null safety

Reply:
IMO, It will still work. You can write
```dart
extension StringExtension on String? {

}
```